src/sentry/eventstream/base.py

```diff
@@ -46,30 +44,18 @@ def _dispatch_post_process_group_task(
         if skip_consume:
             logger.info("post_process.skip.raw_event", extra={"event_id": event.event_id})
         else:
-            random_val = random.random()
             cache_key = cache_key_for_event(
                 {"project": event.project_id, "event_id": event.event_id}
             )
-            if options.get("postprocess.use-cache-key") > random_val:
```

Comment:
@markstory should we deregister the option `postprocess.use-cache-key` as well?

On another note, do you know the current value for this in SaaS? Wondering if it would be safer to set this to 1 before pulling out the code.

Reply:
Currently the option is set to 1 in Saas. I have to talk to single-tenant folks and see if they want to do an incremental rollout there as well.